.github/workflows/test.yml

```diff
@@ -109,5 +113,5 @@ jobs:
       - name: Vet
         run: go vet ./...
       - name: Test
-        run: go test -count=1 ./...
+        run: make test-race
```

Comment:
Here we're running `test-race` as well.

Reply:
That's fine, we're running this on Ubuntu only, which is pretty fast.